test/lit.cfg

```diff
@@ -1327,10 +1327,12 @@ if hasattr(config, 'target_swift_autolink_extract'):
 config.substitutions.append(('%target-swift-modulewrap',
                              config.target_swift_modulewrap))
 
-platform_module_dir = make_path(test_resource_dir, config.target_sdk_name, run_cpu)
+platform_module_dir = make_path(test_resource_dir, config.target_sdk_name)
+if run_vendor != 'apple':
+    platform_module_dir = make_path(platform_module_dir, run_cpu)
 lit_config.note('Using platform module dir: ' + platform_module_dir)
 if test_sdk_overlay_dir is not None:
```

Comment:
While in the area, mind cleaning this up?  It should read: `if test_sdk_overlay_dir:`